src/main/scala/org/allenai/scholar/metrics/metadata/Eval.scala

```diff
@@ -42,17 +43,38 @@ case class Eval(
     * @param idFilter only keep paper ids matching this filter
     */
   def run(
```

Comment:
The output of meta-eval is different.  Instead of the `.tab` file, produce a summary file with overall P/R numbers for each metric, and a 'details' directory, with a file containing the truth and predictions for each metric.

Example of summary file:

    Metric  Precision       Recall
    authorFullName  0.7207056817217243      0.6555228715728715
    authorLastNameNormalized        0.8740543710597181      0.7966786435786484
    titleNonEmpty   1.0     0.958
    titleExact      0.48747390396659707     0.467
    bibliography    0.1270134400667395      0.21940449961211433

Example of details format:

    id      Precision       Recall  Truth   Predicted
    C94-1028        0.5     0.5     Max I Kanovich|Zoya M Shalyapina        Max I Kanovich|Zoya M Shmyapina
    D13-1180        1.0     1.0     Hongbo Chen|Ben He      Hongbo Chen|Ben He
    P11-1008        0.5     0.5     Alexander M Rush|Michael John Collins   Alexander M, Rush|Michael Collins
    W06-1909        1.0     1.0     Sven Hartrumpf  Sven Hartrumpf
    W10-3002        0.8     0.8     Buzhou Tang|Wang XiaoLong|Xuan Wang|Bo Yuan|Shixi Fan   Buzhou Tang|Xiaolong Wang|Xuan Wang|Bo Yuan|Shixi Fanw